x-pack/plugins/security_solution/public/common/components/header_page/index.tsx

```diff
@@ -55,6 +55,14 @@ const Badge = (styled(EuiBadge)`
 ` as unknown) as typeof EuiBadge;
 Badge.displayName = 'Badge';
 
+const HeaderSection = styled(EuiPageHeaderSection)`
+  // Without  min-width: 0, as a flex child, it wouldn't shrink properly
+  // and could overflow its parent.
+  min-width: 0;
+  max-width: 100%;
```

Comment:
Looks like this results in the Rule Details title overflow options (Activate, Edit Settings & overflow menu) being pushed off screen when the main title is truncated. This results in those options not being visible or able to be scrolled to when the title is truncated on lower viewport widths.

Perhaps we can keep this at 100% (for all other pages) and just constrain in the RuleDetails header so that the title overflow options are always visible?

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/123277725-9ba62400-d4c3-11eb-8385-ab0b936874be.gif" />
</p>

Reply:
Verified actually issue is fixed btw, so all good there 👍. Interestingly though, the emoji in the title got converted to a text emoticon after deleting the rule.
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/123278534-41f22980-d4c4-11eb-9852-b93fa95d899f.png" />
</p>

They display fine in the alerts fwiw -- so probably just in how we're parsing out the deleted title from the alert? Not big deal either way, so no need to address here.

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/123278871-90072d00-d4c4-11eb-9d5c-da20bb093d1e.png" />
</p>

Reply:
Must've been on an old commit as just re-tested and not seeing this (the off-screen overflow options hidden) issue. Apologies for the confusion here! 😅